### ISSUE TITLE
Make sure all code snippets use language tags

### ISSUE THREAD

**Issue by Loopiezlol** [enkidevs/curriculum#654]
This also applies to code snippets in **footnotes**, **revisionQuestion**, **practiceQuestion**, not only **content**.

All code snippets should have language tags:

``` -> not cool

```javascript -> cool

At the moment I've applied a hack that will put these by default to `bash` if they lack one. However, we need to fix these,

**Comment by MarvinPollock** [comment]
I've started to insert tags for some of the code snippets and I stumbled on a few, that look like this:

````java
$ jconsole

````
It's inside the Java section, but it's not a Java snippet. Is it still supposed to say java, due to the category it's in or is it supposed to say shell, bash or something like that? :)

**Comment by noahbrenner** [comment]
How should code blocks be tagged that aren't in any programming language? Some examples are:
- Text printed to the console
- Simple diagrams using text

Another special case would be ones like the revision question in the Python insight [Indentation and Commenting](https://github.com/enkidevs/curriculum/blob/master/python/core/console-and-operators/indentation-and-commenting.md#revision). This asks the user to identify the correct commenting syntax, but syntax highlighting would give the answer away rather than relying on the user's understanding of the syntax.

Would `text` be a good "language" to specify for these types of cases?

**Comment by noahbrenner** [comment]
@lizTheDeveloper, any thoughts on the questions that Marvin and I asked here?

**Comment by nem035** [comment]
@noahbrenner, @MarvinPollock there are good points. I would tend to agree that we do need generic `code` block elements that aren't syntax highlighted.

I think we might need to stay with our fork of remark-stringify @Loopiezlol and update the [`code` visitor](https://github.com/remarkjs/remark/blob/master/packages/remark-stringify/lib/visitors/code.js#L45) to stringify with backticks

**Comment by noahbrenner** [comment]
@nem035 it *is* also true that there are a ton of code blocks that *should* have language tags but don't currently.

Would it make any sense to define a specific language tag that means "no highlighting"? That could make the source files easier to lint, since any block without an explicit language tag would clearly be incorrect. There wouldn't be a question every time as to whether a language tag was just forgotten or if it was intentionally omitted.

Since there are a number of existing code blocks marked with `text` as the language, that could be a good choice (if you go that route), since there are a decent number of existing code blocks that already align with that usage.

**Comment by nem035** [comment]
@noahbrenner true, using a designated tag like `text` for this would also work. That also might be the simplest solution. I like it. Let's see what @Loopiezlol and @lizTheDeveloper think about it.

**Comment by noahbrenner** [comment]
Any updates on this topic? It would be great to know the official preferred way of handling these types of code blocks.

**Comment by Mum2541** [comment]
#654

**Comment by Mum2541** [comment]
tex
